doc/nrf/libraries/bluetooth_services/scan.rst

```diff
@@ -1,3 +1,4 @@
+.. _lib_nrf_bt_scan_readme:
 .. _nrf_bt_scan_readme:
 
 Scanning module
```

Comment:
What is the official name of this module, "scanning module" or "BT Scanning Module"?

Reply:
Thank you for your opinion! I was also advised that end users are accustomed to the current module name, so let me keep it as it is for now.